Seminar2/Task3.py

```diff
@@ -0,0 +1,50 @@
+#Напишите программу, которая принимает две строки вида “a/b” - 
+# дробь с числителем и знаменателем. Программа должна возвращать 
+# сумму и произведение* дробей. Для проверки своего кода используйте модуль fractions
+from fractions import Fraction
+
+a = '1/3'
+b = '1/3'
+summ = ""
+# num_a = input("Введите дробь A типа 1/2")
+# num_b = input("Введите дробь B типа 1/2")
+
+index_numerator = 0
+index_denominator = 2
+x1 = int(a[index_numerator])
+x2 = int(a[index_denominator])
+y1 = int(b[index_numerator])
+y2 = int(b[index_denominator])
```

Comment:
Можно сделать значительно проще
```suggestion
x1, x2 = map(int, a.split('/'))
y1, y2 = map(int, b.split('/'))
```

Reply:
А вот умножения я не нашел